projects/element-ng/datatable/index.ts

```diff
@@ -67,7 +99,9 @@ export const SI_DATATABLE_CONFIG: SiDatatableConfig = {
  *
  *  @param configOverrides - overrides that will be merged with the element configuration.
  */
-export const provideSiDatatableConfig = (configOverrides?: INgxDatatableConfig): Provider => ({
+export const provideSiDatatableConfig = (
+  configOverrides?: NgxDatatableConfig | INgxDatatableConfig
```

Comment:
```suggestion
  configOverrides?: NgxDatatableConfig
```

since those types are identical, you can just use the new one